Mathlib/Data/Matrix/Notation.lean

```diff
@@ -72,9 +72,29 @@ end toExpr
 section Parser
 open Lean Elab Term Macro TSyntax
 
+/-- Notation for m×n matrices, aka `Matrix (Fin m) (Fin n) α`.
+
+For instance:
+* `!![a, b, c; d, e, f]` is the matrix with two rows and three columns, of type
+  `Matrix (Fin 2) (Fin 3) α`
+* `!![a, b, c]` is a row vector of type `Matrix (Fin 1) (Fin 3) α` (see also `Matrix.row`).
+* `!![a; b; c]` is a column vector of type `Matrix (Fin 3) (Fin 1) α` (see also `Matrix.col`).
+
+This notation implements some special cases:
+
+* `![,,]`, with `n` `,`s, is a term of type `Matrix (Fin 0) (Fin n) α`
```

Comment:
Not at lean otherwise I'd check, but is this right? I.e. it's `![,,]` and friends, not `!![,,]`?

Reply:
Whoops, this was a mistake!